go/run/scripts/build_docker.sh

```diff
@@ -16,23 +16,25 @@ function build_docker() {
 	# This script currently only supports Linux (or any system that has a working
 	# readlink -e)
 
-	script_name=$1
+	script_name="$1"
```

Comment:
For consistency, could you please quote the other $N arguments when you quote one of them? E.g., it happens that policy_cert ($3) and tao_config ($4) are passed from as paths without spaces, but that's a fragile assumption (one that I made, of course).